swagger.yaml

```diff
@@ -4,8 +4,14 @@ info:
   description: Metatavu Home lambdas
   version: 1.0.0
 
+servers:
+  - url: https://api.severa.visma.com/rest-api
+
```

Comment:
I've not seen this used before in any of our specs and am quite sure it's not necessary.